web/vite.config.ts

```diff
@@ -89,13 +89,6 @@ export default ({ mode }: { mode: any }) => {
       hmr: {
         overlay: true
       },
-      proxy: {
-        "/api": {
-          target: "http://127.0.0.1:7474/",
-          changeOrigin: true,
-          secure: false
-        }
-      }
```

Comment:
This needs to be added back for frontend local development, otherwise we have to recompile frontend every time we make a change.